SQF/dayz_code/medical/transfusion.sqf

```diff
@@ -190,6 +194,7 @@ while {r_doLoop} do {
 	_blood = _unit getVariable ["USEC_BloodQty", 0];
 
 	if (_blood >= r_player_bloodTotal or _bloodAmount == 0) then {
+		if (DZE_transfuseLimit select 0) then { _unit setVariable ["LastTransfusion",time,true]; };
```

Comment:
You should set it locally on the player instead. Then you don't need to do a global setVariable and he can't go around his circle of friends and give each of them a bloodbag.